### PR TITLE
add wiki reference to readme

### PR DESCRIPTION
**Pull request by scubamuc**
as discussed with @pachulo, make PR for adding wiki reference to readme

### CODE REVIEW

**Comment by pachulo** [comment]
What do you think about this @kyrofa ? How we could best point people to our Wiki?

**Comment by kyrofa** [comment]
I'd just put it at the bottom, sort of a "for more information see our wiki".

**Comment by scubamuc** [comment]
> I'd just put it at the bottom, sort of a "for more information see our wiki"

done

File: README.md [hunk 1]
```diff
@@ -225,3 +223,5 @@ And finally, run the tests:
     $ rake test
 
 [1]: https://github.com/nextcloud/nextcloud-snap/wiki/Release-strategy
+
+## See our [Nextcloud snap wiki](https://github.com/nextcloud-snap/nextcloud-snap/wiki) for more information.
```
Comment (pachulo) [inline README.md:227]:
```suggestion
## See our [wiki](https://github.com/nextcloud-snap/nextcloud-snap/wiki) for more information
```

**Review by pachulo — APPROVED** [review]

**Review by Pilzinsel64 — APPROVED** [review]

**Comment by Pilzinsel64** [comment]
Should I merge the commit history?

**Comment by pachulo** [comment]
> Should I merge the commit history?

Yes, please!

**Comment by Pilzinsel64** [comment]
Done.

**Pull request merged**